docs/src/componentDocs/DrawerRailItem/markdown/DrawerRailItemAPIDocs.mdx

```diff
@@ -78,4 +78,20 @@ You can override the default styles used by Brightlayer UI by:
 
 You can render the `<DrawerRailItem>` as a link by setting the `component` prop in the `ButtonBaseProps` to a [`Link`](https://reactrouter.com/web/api/Link) component from `react-router-dom` (recommended), or the native HTML anchor tag `'a'`. This allows you to perform helpful actions such as opening a link in a new browser tab via right-click.
 
-<SharedProps />
+## Shared Props
+
+The following props can be set at any level in the drawer hierarchy ([`<Drawer>`](/components/drawer/api-docs), [`<DrawerBody>`](/components/drawer-body/api-docs), [`<DrawerNavGroup>`](/components/drawer-nav-group/api-docs), [`<DrawerNavItem>`](/components/drawer-nav-item/api-docs), or [`<DrawerRailItem>`](/components/drawer-rail-item/api-docs)). If they are set on a parent, they will be used for all children. For more customization, you can set these props on individual children and they will override any value set on the parent.
```

Comment:
No need to mention for other components , since we are adding specifically for Drawer Rail Item.

Reply:
Changes done, added only for DrawerRailItem.